### ISSUE TITLE
UnusedUses with inline @see false positive

### ISSUE THREAD

**Issue by ostrolucky** [slevomat/coding-standard#658]
```php
use AppBundle\EventSubscriber\Api\ReferrerValidation;

/**
 * Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris eget placerat neque, venenatis gravida lorem. Orc
 * varius @see ReferrerValidation natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
 */
final class Foo
{
}
```

Seems sniff requires `@see` to be at beginning of line, but eg PHPStorm works fine when it's anywhere in the body and it even keeps auto importing functionality working there.

**Comment by kukulich** [comment]
I'm sorry I'll not try to parse description for class names.
